From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:25:39 +0100
Subject: 566076-build_doc_adjustments

Use w3m to build the txt manual.

Gbp-Pq: Topic debian-specific
---
 doc/Makefile.am | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index bc8f856..979f31d 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -109,9 +109,7 @@ uninstall-local:
 
 check:
 manual.txt: manual.html
-	-LC_ALL=C lynx -dump -nolist -with_backspaces -display_charset=us-ascii manual.html > $@ || \
-	LC_ALL=C w3m -dump manual.html > $@ || \
-	LC_ALL=C elinks -dump -no-numbering -no-references manual.html | sed -e 's,\\001, ,g' > $@
+	LC_ALL=C w3m -dump -O UTF-8 manual.html > $@
 
 Muttrc: stamp-doc-rc
 
@@ -123,10 +121,10 @@ stamp-doc-rc: $(top_srcdir)/init.h makedoc$(EXEEXT) $(srcdir)/Muttrc.head
 	touch stamp-doc-rc
 
 manual.html: $(srcdir)/html.xsl $(srcdir)/mutt.xsl stamp-doc-xml $(srcdir)/mutt.css
-	-xsltproc --nonet -o $@ $(srcdir)/html.xsl manual.xml
+	xsltproc --nonet -o $@ $(srcdir)/html.xsl manual.xml
 
 stamp-doc-chunked: $(srcdir)/chunk.xsl $(srcdir)/mutt.xsl stamp-doc-xml $(srcdir)/mutt.css
-	-xsltproc --nonet $(srcdir)/chunk.xsl manual.xml
+	xsltproc --nonet $(srcdir)/chunk.xsl manual.xml
 	touch stamp-doc-chunked
 
 $(CHUNKED_DOCFILES): stamp-doc-chunked
